benchmark/benchmark_level2.sh

```diff
@@ -8,4 +8,16 @@ python benchmark/benchmark.py \
     --slurm-gpus-per-task 1 \
     --slurm-ntasks 1 \
     --slurm-total-cpus 12 \
-    --slurm-template-path benchmark/trl.slurm_template
\ No newline at end of file
+    --slurm-template-path benchmark/trl.slurm_template
+
+# compound: Cerebras-GPT-6.7B + deepspeed zero2 + grad_accu
+python benchmark/benchmark.py \
+    --command "accelerate launch --config_file examples/accelerate_configs/deepspeed_zero2.yaml examples/scripts/sentiment_tuning.py --ppo_config.exp_name sentiment_tuning_Cerebras-GPT-6.7B_grad_accu_deepspeed_stage2  --ppo_config.batch_size 32  --ppo_config.mini_batch_size 32 --ppo_config.log_with wandb --ppo_config.model_name cerebras/Cerebras-GPT-6.7B --ppo_config.reward_model sentiment-analysis:cerebras/Cerebras-GPT-6.7B" \
```

Comment:
Eventually I think we should do the "proper" thing and fine-tune these models on IMDB so we have a genuine good policy / reward model. Of course, not necessary for this PR, but perhaps good to be as realistic as possible for the benchmark

Reply:
I think that sounds good. Perhaps we can set up an end-to-end example where we train the reward model and then the policy model at the same time.